plugins/wasm_bpf/func-close-bpf-object.cpp

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2019-2022 Second State INC
+
+#include "func-close-bpf-object.h"
+#include <shared_mutex>
+
+namespace WasmEdge {
+namespace Host {
+
+Expect<int32_t> CloseBpfObject::body(const WasmEdge::Runtime::CallingFrame &,
+                                     handle_t program) {
+  std::shared_lock guard(this->state->lock);
```

Comment:
You don't need to write this. Do you have the same name as the variables as the state in this scope?